CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Snyk Security Changelog
 
+## [2.7.15]
```

Comment:
Next Tuesday is 2.7.13, so you should merge your stuff into that version.

Reply:
I don't think so. But basically, each Tuesday we release, and it's automatically incrementing the patch part. The changelog versions are automatically linked into the description in the Jetbrains Marketplace (what's new) and added to the github release. So the new version increment and the description in the Changelog should align.